app/javascript/submission_handling/renew_checkout.js

```diff
@@ -18,6 +20,38 @@ const updateCheckout = function (data) {
     }
 };
 
+const renderProcessing = function () {
+    const infoHTML = `<div class="alert alert-info" role="alert">
+                        <span>Renewals are in progress...</span>
+                      </div>`;
+
+    if (document.querySelector('.renewals-summary')) {
+        document.querySelector('.renewals-summary').innerHTML = infoHTML;
```

Comment:
so this condition is when there is a stale (complete or even active) summary notification up and it'll get replaced/over-written by this new renewal batch action. right?

Reply:
yeah, correct.

Reply:
ok cool. do think the renaming of the function will help set understanding for future code readers